### DISCUSSION TITLE
Ultimate SD Upscale

### DISCUSSION THREAD

**Discussion by toyssamurai** [lllyasviel/stable-diffusion-webui-forge#543] [Q&A]
I installed Ultimate SD Upscale. I don't have problem running it but once an image is generated, it's not automatically saved. I have to manually press the Save icon. What should I change in Settings?

**Comment by staticsys** [comment]
This image is the settings I have mine as, I haven't had any issues with it.
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/157920682/bb13ad3d-39ca-43ce-839f-f39002677332)

**Comment by trevormads-a11y** [comment]
Upscale the image.
